env/includes/settings/tk-flame.yml

```
@@ -30,8 +30,8 @@ settings.tk-flame.project:
     tk-multi-loader2: "@settings.tk-multi-loader2.flame"
   location: "@engines.tk-flame.location"
   context_menu: [
-    { name : "Load...", app_instance: "tk-multi-loader2", display_name: "Shotgun Loader..."},
-    { name : "Shotgun Panel...", app_instance: "tk-multi-shotgunpanel", display_name: "Shotgun Panel..."}
+    { name : "Load...", app_instance: "tk-multi-loader2", display_name: "ShotGrid Loader..."},
+    { name : "ShotGrid Panel...", app_instance: "tk-multi-shotgunpanel", display_name: "ShotGrid Panel..."}
```

```suggestion
    { name : "Shotgun Panel...", app_instance: "tk-multi-shotgunpanel", display_name: "ShotGrid Panel..."}
```
I'm pretty sure that change broken the app. "Shotgun Panel..." is the name of the command to run in Toolkit and app commands shouldn't have been renamed during the rebranding proces